### PR TITLE
feat(mirinae): add PCenteredLayout and PCenteredLayoutHeader components

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [x] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [ ] Apps
  - [x] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

image
image
image

image
image

### CODE REVIEW

File: packages/mirinae/src/layouts/centered-layout/assets/background.png
Comment (skdud4659) [inline packages/mirinae/src/layouts/centered-layout/assets/background.png:1]: 👻 bruise

**Review by skdud4659 — APPROVED** [review]
LGTM!
Thank you for your hard work👍🏻

**Review by dsict — APPROVED** [review]
Nice! Thank you always.

File: packages/mirinae/src/layouts/centered-layout/centered-layout-header/PCenteredLayoutHeader.vue [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    console.error('[Centered Layout Header] when showStep is true, currentStep and totalSteps must be number');
+    return false;
+});
+
+const handleClickCloseButton = () => {
+    emit('close');
+};
+</script>
+
+<template>
```
Comment (yuda110) [inline packages/mirinae/src/layouts/centered-layout/centered-layout-header/PCenteredLayoutHeader.vue:33]: wow! The template is below!

**Review by yuda110 — APPROVED** [review]
LGTM!!!!

File: packages/mirinae/src/layouts/centered-layout/centered-layout-header/PCenteredLayoutHeader.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+## Basic
+
+<Canvas>
+    <Story name="Basic">
+        {{
+            components: {
+                PCenteredLayoutHeader
+            },
+            template: `
+            <div style="width: 360px">
```
Comment (sulmoJ) [inline packages/mirinae/src/layouts/centered-layout/centered-layout-header/PCenteredLayoutHeader.stories.mdx:51]: Suggestion: Personally, I would like it if the width was 100%. It looks like there is a fixed size.

image

image

**Review by sulmoJ — APPROVED** [review]
LGTM!
I left a suggestion.
There was an inconvenience when I used it as a console component before, and I think I need to double-check it after the subsequent modifications.
(It seems like it will take too long to confirm it now... haha)

**Pull request merged**